tools/cli/src/templates/config/schemas/teacher.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Domain specific schema can go to respetive repository like https://github.com/ref-registries/edu-core-registries

Reply:
This is just a sample/placeholder schema for the getting started guide. When creating a registry, the user is asked for the path to the schemas they actually want to use. If they do not enter a path, this minimal placeholder schema is used.

I could add it to https://github.com/ref-registries/edu-core-registries as well if that helps.

Would you still suggest I move it or just make a copy?

Reply:
There is one already, how different is this one compared to that? just trying to avoid multiple instances of the same.

Reply:
The schema configuration in https://github.com/ref-registries/edu-core-registries contains a lot more entities and fields as compared to the minimal schemas in the CLI. 

The https://github.com/ref-registries/edu-core-registries schemas are more representative for the education domain (includes lots of fields such as AcademicExperience, Qualifications, IdentityDetails), while the CLI schemas are extremely minimal and just contain name, phone, email, subject and school fields that help demonstrate the registry's core features, e.g., create/invite, retrieve, update, claim, attest and consent.